README.md

```diff
@@ -57,61 +57,59 @@ endpoints, and define and manage a rich set of security policy.
-
-With each of these tutorials we provide details for running the demonstration 
-using manual setup on your own servers, or with a quick set-up in a virtualized
-environment using Vagrant, or a number of cloud services.
-
-We also provide the following additional demonstrations: 
-- [Docker Swarm cluster](docs/getting-started/powerstrip/CalicoSwarm.md) 
-- [Calico and Kubernetes](docs/kubernetes/README.md)
-
-
-### Networking options
-
-#### Docker default networking
-
-This uses Dockers standard networking infrastructure, requiring you to 
-explicitly add a created container into a Calico network.
-
-This is compatible with all Docker versions from 1.6 onwards.
-
-#### Docker with libnetwork
-
-Docker's native [libnetwork network driver][libnetwork] is available in the 
-Docker 1.9 release currently underoing development.
-
-Setup of the libnetwork environment is a little more involved since it requires
-the current master (1.9.dev) builds of Docker, and the use of etcd as a
-datastore for Docker clustering.
-
-#### Docker with Powerstrip
-
-[Powerstrip][powerstrip] is a pluggable HTTP proxy for the Docker API. Development 
-of the Powerstrip version of Calico is lagging behind the master branch, so an 
-older version of `calicoctl` and the `calico-node` docker image are required.
-  
-## FAQ 
-For more information on what you can do with Calico, please visit the 
-[frequently asked questions](docs/FAQ.md) page. 
+To get started using Calico, we recommend running through one or more of the 
+available demonstrations from our [Getting Started](getting-started/README.md) 
+guides.
+
+These guides will help you understand the different networking options when 
+using Calico by allowing you to configure and run a Calico cluster both locally 
+and with your preferred cloud service.
+
+If you want to jump directly into a guide ***right now***, we recommend using 
+one of our automated guides, such as:
+
+  - [Vagrant Ubuntu](docs/getting-started/VagrantUbuntu.md) or 
+    [Vagrant CoreOS](docs/getting-started/VagrantCoreos.md) for local installs 
+    with VirtualBox
+  - [GCE](docs/getting-started/GCE.md), [AWS](docs/getting-started/AWS.md) or 
+    [DigitalOcean](docs/getting-started/DigitcalOcean.md) for Calico with cloud 
+    services.
+
+Also check out the [Calico Kubernetes integration](kubernetes/README.me) to 
+learn how to configure Calico with the Kubernetes Orchestrator.
+
+## Documentation
+
+The Calico documentation provides information for both Calico users and 
+developers.
+
+  - **Learn how to configure Calico features in a deployment**
+    - [BGP Configuration](docs/bgp.md) for managing global, node-to-node, and 
+      node-specific BGP peers
+    - [Logging Configuration](docs/logging.md) to set logging levels and choose 
+      where Calico logs should be stored
+    - [Advanced Network Policy](docs/AdvancedNetworkPolicy.md) to configure 
+      security policy between Calico nodes and other networks

```

Comment:
"...configure network policy for your containers including connections to other containers and external networks."

Avoids too much of our specific vocab in this first intro document.